vue/components/ui/organisms/header/header.vue

```diff
@@ -301,13 +356,22 @@ export const Header = {
         apps: {
             type: Object,
             default: () => ({})
+        },
+        notifyColor: {
+            type: String,
+            default: "#3c80cd"
+        },
+        announcements: {
+            type: Object,
+            default: {}

```

Comment:
Optionals -> use `null`. Semantically, it's a little bit different to have something not defined or having it defined and empty.